conanfile.py

```diff
@@ -40,6 +47,7 @@ def _configure_cmake(self):
         #self._cmake.definitions["ENABLE_DYNAMIC_LOADING"] = True
         self._cmake.definitions["BUILD_SHARED_LIBS"] = False
         #self._cmake.definitions[""] =
+        self._cmake.definitions['CMAKE_POSITION_INDEPENDENT_CODE'] = self.options.fPIC
```

Comment:
You need to add something like 
`if self.settings.os != "Windows":`
before the CMake definition, because you have deleted that option for windows.

Reply:
Ah, thanks! Also, you are right that I should check for `os` rather than `compiler`, since code is always relocatable on Windows – it has nothing to do with the compiler. (I just copied the compiler check from the Bincrafters howto.)